automation/src/test/scala/org/broadinstitute/dsde/test/api/AuthDomainGroupApiSpec.scala

```diff
@@ -59,10 +67,11 @@ class AuthDomainGroupApiSpec extends FreeSpec with Matchers with WorkspaceFixtur
               // Note: this is not a passthrough to Rawls is because library needs to overwrite any publish and discoverableByGroups values
               Orchestration.workspaces.clone(projectName, workspaceName, projectName, workspaceCloneName, Set(authDomainName))(user.makeAuthToken())
 
-              // the authdomain group should be found in cloned workspace
-              val groups = Rawls.workspaces.getAuthDomainsInWorkspace(projectName, workspaceCloneName)(user.makeAuthToken())
-              groups should contain theSameElementsAs List(authDomainName)
-
+              eventually {
+                // the authdomain group should be found in cloned workspace
+                val groups = Rawls.workspaces.getAuthDomainsInWorkspace(projectName, workspaceCloneName)(user.makeAuthToken())
+                groups should contain theSameElementsAs List(authDomainName)
```

Comment:
` groups should contain theSameElementsAs List(authDomainName)` should be outside the eventually

Reply:
Moved `eventually` block to be inside `AuthDomainMatcher`. See [workbench-lib PR 224](https://github.com/broadinstitute/workbench-libs/pull/224)